### PR TITLE
Create issue_template.md

### PR DESCRIPTION
**Pull request by hmarchadour**
Create an issue template.

### CODE REVIEW

**Review by mbats — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+### Description
+
+...
+
+### Steps to reproduce
+
+...
+
+### Expected behavior
```
[the remark this answers is not in the document]
Comment (hmarchadour) [reply .github/ISSUE_TEMPLATE/issue_template.md:12]: I think an "Actual behavior" section could replace the "Description".
Indeed, it's an issue template, it means that the actual behavior has an error/a strange behavior.
Make a distinction between the Description and the Actual behavior seems hard to me.

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+* [ ] **I have checked that this bug has not yet been reported by someone else.**
+* [ ] **I have checked that this bug appears on Chrome.**
+
```
[the remark this answers is not in the document]
Comment (hmarchadour) [reply .github/ISSUE_TEMPLATE/issue_template.md:3]: right

**Comment by hmarchadour** [comment]
I will recreate a pull request. The eca check seems to fail since I've make the first commit with the wrong email....

**Comment by sbegaudeau** [comment]
Ok, I'll will close this pull request then.

**Pull request closed without merging**